### ISSUE TITLE
1.0 release

### ISSUE THREAD

**Issue by mbajur** [foca/granola-rails#3]
Any plans to release 1.0? :)

**Comment by foca** [comment]
Yes! I haven't been working with rails lately so it's been a bit on the back burner. Have you had any issues using granola with rails? Which versions (of ruby, rails, granola) are you using?

**Comment by mbajur** [comment]
I didn't even tried as dependencies are too restrictive to try. I was using a raw non-rails granola

**Comment by foca** [comment]
I was *sure* I had updated granola-rails to depend on granola 1.0. My bad! :(

I've cut a 0.2.0 release that depends on Granola 1.0 and should work with any stable rails version. Please let me know if this works fine. When I have some time I'll attack the other open issue and will then release 1.0.

(Obviously PRs and suggestions are welcome! 😄)

**Comment by mbajur** [comment]
Great! Will try it in upcoming days and let you know :)
